hydroflow/examples/graph_reachability/README.md

```diff
@@ -1,8 +1 @@
-# Graph Reachability
-
-To run:
-```
-cargo run -p hydroflow --example graph_reachability
-```
-
-Adding the `-- --graph <graph_type>` flag to the end of the command line above will print out a node-and-edge diagram of the program. Supported values for `<graph_type>` include [mermaid](https://mermaid-js.github.io/) and [dot](https://graphviz.org/doc/info/lang.html).
```

Comment:
Is this intentional? seems like the deleted version should stick around. maybe accident from refactoring in the surface syntax version